default/web_tt2/report_abuse.tt2

```diff
@@ -0,0 +1,5 @@
+<!-- report_abuse.tt2 -->
+[% IF conf.show_report_abuse %]
+    [% | mailto("${conf.listmaster_email}@${domain}",{subject=>"Sympa abuse report: ${list}"}) %][%|loc%]Report abuse[%END%][%END%]
+[% END %]
+<!-- end report_abuse.tt2 -->
```

Comment:
The subject `Sympa abuse report: ${list}` would be better to be translated.

Reply:
Mmh… not sure: on Framalistes (french instance), we have german-speaking people using our instance. I would receive a mail with a subject in german, and so would not give it with all the attention it deserves + there's a risk that I may think it's a spam. 

I'm open to discussion, though. Maybe translating it to the instance default language? Can I force the translation to a given language? If yes, how?

Reply:
I feel it making sense that we stick to single subject (language) so that we won't miss the report.  If it would be translated, language of domain that listmaster belongs to would be used.

However, currently it is difficult for templates to choose language differ from list's lang.  Maybe sticking to english is reasonable.

Additonally, I found that [syntax of mailto URL](https://tools.ietf.org/html/rfc6068#section-2) containing non-ASCII parameters can be complex and it might mess several mailers.  Limiting characters in subject to ASCII looks reasonable.

Reply:
I agree.